style/landcover.mss

```diff
@@ -261,20 +262,6 @@
     }
   }
 
-  [feature = 'leisure_dog_park'] {
-    [zoom >= 10] {
-      polygon-fill: @leisure;
-      [way_pixels >= 4]  { polygon-gamma: 0.75; }
-      [way_pixels >= 64] { polygon-gamma: 0.3;  }
-    }
-    [zoom >= 16] {
-      polygon-pattern-file: url('symbols/dog_park.png');
-      polygon-pattern-alignment: global;
-      [way_pixels >= 4]  { polygon-pattern-gamma: 0.75; }
-      [way_pixels >= 64] { polygon-pattern-gamma: 0.3;  }
-    }
-  }
-
```

Comment:
Removes pattern fill from dog parks.   Polygon fill is already handled elsewhere.